frontend/src/components/print/print-react/components/picasso/longestTime.js

```diff
@@ -0,0 +1,9 @@
+import dayjs from '@/common/helpers/dayjs.js'
+
+export default function longestTime(times) {
```

Comment:
Because the last time is the longest time?
This is different depending on the locale?

Reply:
Yes, depending on the locale, and in the future also depending on a dynamically adjusted times array (depending on the schedule entries in the period).